lms/urls.py

```diff
@@ -33,6 +33,9 @@
     url(r'', include('student.urls')),
     # TODO: Move lms specific student views out of common code
     url(r'^dashboard$', 'student.views.dashboard', name="dashboard"),
+
+    url(r'^course_refund_status/$', 'student.views.course_refund_status', name="course_refund_status"),
```

Comment:
I think the URL can be `/course/<course_id>/refund_status`. This way, the api looks clearer about the intention